CommonClasses/interface/DiPhotonInfo.h

```diff
@@ -24,7 +25,7 @@ namespace ExoDiPhotons
     bool isEEEE;
   };
 
-  std::string diphotonBranchDefString("Minv/D:qt:deltaPhi:deltaEta:deltaR:cosThetaStar:cosThetaStar_old:isEBEB/O:isEBEE:isEEEB:isEEEE");
+  std::string diphotonBranchDefString("Minv/D:qt:deltaPhi:deltaEta:deltaR:cosThetaStar:cosThetaStar_old:chiDiphoton:isEBEB/O:isEBEE:isEEEB:isEEEE");
```

Comment:
Please change this to:
``std::string diphotonBranchDefString("Minv/D:qt:deltaPhi:deltaEta:deltaR:cosThetaStar:cosThetaStar_old:isEBEB/O:isEBEE:isEEEB:isEEEE:chiDiphoton/D");``
To maintain backwards compatibility with existing ntuples, the ordering of existing leafs of the branch should not be changed. The ordering of the variables within the `diphotonInfo_t` struct should be changed to match this branch definition as well.